Integrations/integration-QRadar.yml

```diff
@@ -42,296 +38,554 @@ configuration:
+        enrich_offense_result(res)
+        res = filter_dict_nonintersection_key_to_value(replace_keys(res, OFFENSES_NAMES_MAP), OFFENSES_NAMES_MAP)
+        return get_entry_for_object('QRadar offenses', res, CUSTOM_HEADERS, 'QRadar.Offense(val.ID === obj.ID)')
+
+    def get_offense_by_id_command():
+        offense_id = demisto.args().get('offense_id')
+        res = send_get_offense_by_id_request(offense_id, demisto.args().get('filter'), demisto.args().get('fields'))
+        enrich_offense_result(res, full_enrichment=True)
+        res = filter_dict_nonintersection_key_to_value(replace_keys(res, SINGLE_OFFENSE_NAMES_MAP), SINGLE_OFFENSE_NAMES_MAP)
+        return get_entry_for_object('QRadar Offenses', res, CUSTOM_HEADERS, 'QRadar.Offense(val.ID === obj.ID)')
+
+    def update_offense_command():
+        args = demisto.args()
+        if 'closing_reason_name' in args:
+            args['closing_reason_id'] = convert_closing_reason_name_to_id(args.get('closing_reason_name'))
+        elif 'CLOSED' == args.get('status'):
+            raise ValueError('Invalid input - must provide closing reason name (may use "qradar-get-closing-reasons" command to get them) to close offense')
+        offense_id = args.get('offense_id')
+        res = send_update_offense_request(offense_id)
+        enrich_offense_result(res, full_enrichment=True)
+        res = filter_dict_nonintersection_key_to_value(replace_keys(res, SINGLE_OFFENSE_NAMES_MAP), SINGLE_OFFENSE_NAMES_MAP)
+        return get_entry_for_object('QRadar Offense', res, CUSTOM_HEADERS, 'QRadar.Offense(val.ID === obj.ID)')
+
+    def search_command():
+        res = send_search_request(demisto.args())
+        res = filter_dict_nonintersection_key_to_value(replace_keys(res, SEARCH_ID_NAMES_MAP), SEARCH_ID_NAMES_MAP)
+        return get_entry_for_object('QRadar Search', res, CUSTOM_HEADERS, 'QRadar.Search(val.ID === obj.ID)')
+
+    def get_search_command():
+        res = send_get_search_request(demisto.args().get('search_id'))
+        res = replace_keys(res, SEARCH_ID_NAMES_MAP)
+        return get_entry_for_object('QRadar Search Info', res, CUSTOM_HEADERS, 'QRadar.Search(val.ID === obj.ID)')
+
+    def get_search_results_command():
+        search_id = demisto.args().get('search_id')
+        res = send_get_search_results_request(search_id)
+        return get_entry_for_object('QRadar Search Results', res, CUSTOM_HEADERS, 'QRadar.Search(val.ID === {0}).Result'.format(search_id))
+
+    def get_assets_command():
+        res = send_get_assets(demisto.args().get('range'), demisto.args().get('filter'), demisto.args().get('fields'))
+        res, human_readable_res = create_assets_result(res)
+        return get_entry_for_assets('QRadar Assets', res, human_readable_res, CUSTOM_HEADERS)
+
+    def get_asset_by_id_command():
+        _filter = "id=" + str(demisto.args().get('asset_id'))
+        res = send_get_assets(_filter=_filter)
```

Comment:
you don't need to write `send_xxx`, we just do `get_assets`

Reply:
and get_assets should usually return `assets`, so it should be 
`assets = get_assets(...)`
or 
`raw_assets = get_assets(...)`